generic/tests/vlan.py

```diff
@@ -155,6 +155,20 @@ def flood_ping(src, dst):
         vm_.verify_alive()
 
     for vm_index, vm in enumerate(vms):
+        if params["os_type"] == "windows":
+            session = vm.wait_for_serial_login(timeout=login_timeout)
+            win_vlan_id = int(params.get("win_vlan_id", 900))
+            set_vlan_cmd = params["set_vlan_cmd"]
+            nicid = utils_net.get_windows_nic_attribute(session=session,
+                                                        key="netenabled", value=True,
+                                                        target="netconnectionID")
+            session.cmd(set_vlan_cmd % nicid)
+            ifname.append(nicid)
+            dev_mac = vm.virtnet[0].mac
+            vm_ip.append(utils_net.get_guest_ip_addr(session, dev_mac, os_type="windows"))
+            session_ctl.append(session)
+            continue
```

Comment:
~~`continue`~~

Reply:
Windows guest didn't need to do following steps, so add `continue` here...

Reply:
sorry， my fault.